integrations/ultralytics-yolov5/tutorials/sparse-transfer-learning.md

```diff
@@ -0,0 +1,379 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Sparse Transfer Learning
+
+This page explains how to fine-tune a pre-sparsified YOLOv5 model with SparseML's CLI.
+
+## Overview
+
+Sparse Transfer is quite similiar to the typical YOLOv5 training, where we fine-tune a checkpoint pretrained on COCO onto a smaller downstream dataset. However, with Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified YOLOv5 and maintain sparsity while the training process occurs.
+
+[SparseZoo](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=detection&page=1) contains pre-sparsified checkpoints for each version of YOLOv5, which can be used as the starting checkpoints for the training process.
+
+## Installations
+
+Install via `pip`:
+
+```
+pip install sparseml[torchvision]
+```
+
+## Sparse Transfer Learning with VOC
+
+Let's try a simple example of Sparse Transfer Learning YOLOv5s onto the VOC dataset.
+
+Run the following:
+```
+sparseml.yolov5.train \
+  --weights zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned75_quant-none?recipe_type=transfer_learn \
+  --recipe zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned75_quant-none?recipe_type=transfer_learn \
+  --data VOC.yaml \
+  --patience 0 \
+  --cfg yolov5s.yaml \
+  --hyp hyps/hyp.finetune.yaml
+```
+
+Lets disucss the key arguments:
+- `--weights` specifies the starting checkpoint for the training process. Here, we passed a SparseZoo stub, which
+identifies the 75% pruned-quantized YOLOv5s model in the SparseZoo. The script downloads the PyTorch model to begin training. In addition to SparseZoo stubs, you can also pass a local path to a PyTorch checkpoint.
+
+- `--recipe` specifies the transfer learning recipe. Recipes are YAML files that declare the sparsity related algorithms
+ that SparseML should apply. For transfer learning, the recipe instructs SparseML to maintain sparsity during training
+ and to apply quantization over the final epochs. In this case, we passed a SparseZoo stub, which instructs SparseML
```

Comment:
In this case, we passed a SparseZoo stub instructing SparseML to download a premade YOLOv5s transfer learning recipe.